### PR TITLE
15578 versioning

### PR DESCRIPTION
**Pull request by damonabenson**
Updates for notice.txt from Legal team

### CODE REVIEW

File: packages/node/package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
   "dependencies": {
     "@progress/jsdo-core": "^6.0.0",
     "base-64": "^0.1.0",
+    "node-localstorage": "^1.3.1",
```
Comment (edselg) [inline packages/node/package.json]: We removed the requirement the node-localstorage module. It is not expected to be in the latest package.json.

File: src/auth/progress.auth.form.js [hunk 2]
```diff
@@ -118,10 +101,7 @@ limitations under the License.
                 xhr.onreadystatechange = function () {
                     if (xhr.readyState === 4) {
                         // process the response from the Web application
-                        try {
```
Comment (nicoletteschreier) [inline src/auth/progress.auth.form.js]: what is this code change?

File: packages/node/src/loaddep.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /*global XMLHttpRequest:true, localStorage: true, sessionStorage: true, btoa:true*/
 /*jslint nomen: true*/
 
-(function () {
```
Comment (nicoletteschreier) [inline packages/node/src/loaddep.js]: what are the changes in this file?

File: src/auth/progress.auth.js [hunk 4]
```diff
@@ -1,218 +1,139 @@
...
+            xhr,
+            uriForRequest,
+            header,
+            that = this;
 
-        return authProv;
-    };
-        
-
-    // ADD METHODS TO THE AuthenticationProvider PROTOYPE
```
Comment (nicoletteschreier) [inline src/auth/progress.auth.js]: it looks like there are large code changes being made to this file. what is going on here?

File: packages/angular/package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@progress/jsdo-angular",
-  "version": "6.0.1",
+  "version": "6.0.0",
```
Comment (nicoletteschreier) [inline packages/angular/package.json]: here is a 6.0.0

File: packages/core/package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@progress/jsdo-core",
-  "version": "6.0.1",
+  "version": "6.0.0",
```
Comment (nicoletteschreier) [inline packages/core/package.json]: 6.0.0?

File: packages/nativescript/package.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@progress/jsdo-nativescript",
-  "version": "6.0.1",
+  "version": "6.0.0",
```
Comment (nicoletteschreier) [inline packages/nativescript/package.json]: 6.0.0

**Pull request closed without merging**